alibi/explainers/tests/test_simiarlity/test_grad_methods_integration.py

```diff
@@ -303,3 +311,62 @@ def test_multiple_test_instances_stored_grads_asym_dot(precompute_grads):
     scores = np.array([[50, 50, 2*50**2]], dtype=np.float32) / (denoms + 1e-7)
     scores.sort()
     np.testing.assert_allclose(explanation.scores, scores[:, ::-1], atol=1e-4)
+
+
+@pytest.mark.parametrize('linear_cls_model',
+                         [
+                             ({'framework': 'pytorch', 'input_shape': (10,), 'output_shape': 10, 'batch_norm': True}),
+                             ({'framework': 'tensorflow', 'input_shape': (10,), 'output_shape': 10, 'batch_norm': True})
+                         ],
+                         indirect=True, ids=['torch-model', 'tf-model'])
+def test_no_warnings_batch_norm(linear_cls_model):
+    """
+    Test that no warnings are raised when using batch norm layers.
```

Comment:
Think need to add another sentence for future developers as not sure I follow the reasoning why specifically batch norm layers shouldn't trigger warnings.